tool-plugins/vscode/src/core/extension.ts

```diff
@@ -117,17 +122,24 @@ export class BallerinaExtension {
                     return Promise.reject(msg);
                 }
             }
-            log("Using " + this.ballerinaHome + " as the Ballerina home.");
             // Validate the ballerina version.
             const pluginVersion = this.extension.packageJSON.version.split('-')[0];
             return this.getBallerinaVersion(this.ballerinaHome, this.overrideBallerinaHome()).then(ballerinaVersion => {
                 ballerinaVersion = ballerinaVersion.split('-')[0];
                 log(`Plugin version: ${pluginVersion}\nBallerina version: ${ballerinaVersion}`);
                 this.checkCompatibleVersion(pluginVersion, ballerinaVersion);
+
+                // versions less than 1.0.2 are incapable of handling cli commands for langserver and debug-adapter
+                this.isNewCLICmdSupported = this.compareVersions(ballerinaVersion, "1.0.3", true) >= 0;
```

Comment:
logic should be fixed so that only versions that are >= 1.1.0 supported in new approach

Reply:
Yes. need to update the version into 1.1.0. Added the change.